src/channels/receiver.rs

```diff
@@ -253,6 +255,38 @@ where
         })
     }
 
+    ///
+    /// Sends message over chromecast message bus
+    ///
+    /// Receiver can observe messages using `context.addCustomMessageListener` with custom namespace.
+    ///
+    ///```javascript, no_run
+    /// context.addCustomMessageListener('urn:x-cast:com.example.castdata', function(customEvent) {
+    /// // do something with message
+    /// });
+    ///```
+    ///
+    /// Namespace should start with `urn:x-cast:`
+    ///
+
```

Comment:
__nit:__ just for the consistency (and rustdoc)
```suggestion
    /// Sends a message over a cast device's message bus.
    ///
    /// Receiver can observe messages using `context.addCustomMessageListener` with a custom namespace.
    ///
    /// # Examples
    ///
    /// ```javascript, no_run
    /// context.addCustomMessageListener('urn:x-cast:com.example.castdata', function(customEvent) {
    ///   // do something with message
    /// });
    /// ```
    /// # Arguments
    ///
    /// * `namespace` - Message namespace that should start with `urn:x-cast:`.
    /// * `message` - Message instance to send.
```

Reply:
Will update